### PR TITLE
(maint) update changelog for 0.4 release

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

**Comment by underscorgan** [comment]
There's a bunch of other stuff that's been merged ...https://github.com/puppetlabs/ezbake/compare/0.3.25...master

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.3.26 - 2016-05-11
```
Comment (haus) [inline CHANGELOG.md]: Is systemd support for debian not featureful? 😄

**Comment by underscorgan** [comment]
Heeeey @cprice404 @jpinsonault want to chime in on the EZ-72 stuff and if it's ready for release

**Comment by haus** [comment]
@McdonaldSeanp for a 0.4.0 release, you'll also want to tweak project.clj to move the version to "0.4.0-SNAPSHOT"

**Comment by McdonaldSeanp** [comment]
ty

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## 0.4.0 - 2016-05-11
+  * Add Ubuntu 15.10 (Wily) to default build targets
+  * Add Ubuntu 16.04 (Xenial) to default build targets
+  * Add Debian systemd script support
+  * Add split bootstrap functionality to allow for user-configurable service
+  entries alongside services that shouldn't be modified.
+  * Changed folder structure for staging to add an ezbake/system-config dir
+
+
```
Comment (underscorgan) [inline CHANGELOG.md]: [nitpick] one blank line here would be consistent with the rest of the file

**Comment by jpinsonault** [comment]
This looks good to me 👍

**Comment by geoffnichols** [comment]
👍

**Pull request merged**